test/helpers.test.js

```diff
@@ -57,6 +61,23 @@ describe('helpers', function() {
       testValidationRejectsValue(validateName, 'm.prop');
     });
   });
+
+  // test checkRelationName()
+  describe('checkRelationName()', function() {
+    var firstModelDefinition = new ModelDefinition([
+      {name: 'name', id: 1},
+      {name: 'city', id: 2}
+    ]);
+    var secondModelDefinition = new ModelDefinition([]);
```

Comment:
This second model definition looks redundant to me. The algorithm does not depend on the number of model properties, so why is a model with no properties so special that it deserves an explicit test?